src/apify_client/clients/resource_clients/task_collection.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Dict, Optional
+
+from ..._utils import _filter_out_none_values_recursively
+from ..base.resource_collection_client import ResourceCollectionClient
+
+
+class TaskCollectionClient(ResourceCollectionClient):
+    """Sub-client for manipulating tasks."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the TaskCollectionClient with the passed arguments."""
+        super().__init__(*args, resource_path='actor-tasks', **kwargs)
+
+    def list(self, *, limit: Optional[int] = None, offset: Optional[int] = None, desc: Optional[bool] = None) -> Dict:
+        """List the available tasks.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-collection/get-list-of-tasks
+
+        Args:
+            limit: How many tasks to retrieve
+            offset: What tasks to include as first when retrieving the list
+            desc: Whether to sort the tasks in descending order based on their modification date
+
+        Returns:
+            The list of available tasks matching the specified filters.
+        """
+        return self._list(limit=limit, offset=offset, desc=desc)
+
+    def create(
+        self,
+        *,
+        actor_id: str,
+        name: str,
+        build: Optional[str] = None,
+        timeout_secs: Optional[int] = None,
+        memory_mb: Optional[int] = None,
+        task_input: Optional[Dict] = {},
+    ) -> Dict:
+        """Create a new task.
+
+        https://docs.apify.com/api/v2#/reference/actor-tasks/task-collection/create-task
+
+        Args:
+            actor_id (string): Id of the actor that should be run
+            name (string): Name of the task
+            build (str, optional): Specifies the actor build to run. It can be either a build tag or build number.
+                                   By default, the run uses the build specified in the task settings (typically latest).
+            memory_mb (int, optional): Memory limit for the run, in megabytes. By default, the run uses a memory limit specified in the task settings.
```

Comment:
memory_mbytes? (as in the TaskClient)

Reply:
I think that from `_mb` suffix it should be sufficiently clear that it means `megabytes`, isn't it?

Reply:
It should be consistent with the API though. And API uses `mbytes`. We already had different names between API and client v0. It was not working well at all.

Reply:
Ok, I renamed to `mbytes`